src/components/StyledLink.tsx

```diff
@@ -1,12 +1,14 @@
 import React from 'react';
 import { Link, LinkProps } from 'react-router-dom';
+import classNames from 'classnames';
 
 type StyledLinkProps = LinkProps & React.RefAttributes<HTMLAnchorElement>;
 
 const StyledLink: React.FC<StyledLinkProps> = ({ children, className, ...rest }) => {
-  const linkClassName =
-    className ?? 'text-sp-600 hover:text-sp-700 font-semibold hover:underline text-base tracking-wide';
-
+  const linkClassName = classNames(
```

Comment:
what does the classNames method do over something like this? linkClassName = "text-sp-600 hover ... " + className. 
Also this change removes the flexibility of styling the link as you cant override the defaults. Not sure this is an issue if we want all StyledLinks to look the same

Reply:
The code seems bit different in mine _**const linkClassName = classNames(
    'text-sp-600 hover:text-sp-700 font-semibold hover:underline text-base tracking-wide',
    className
  );**_

Reply:
The end result is same as you specified but I think  it does few more extra checks. The classNames function takes any number of arguments which can be a string or object. For example, If the value associated with a given key is falsy, that key won't be included in the output. So, it is more like a convenience method especially  if there are multiple strings.

Reply:
Ah ok. Sorry I poorly explained it, my code looks the same as yours but I was just wondering the advantages of classNames over just extending the string, but as you've just explained its fine.